sdk/trafficmanager/Azure.ResourceManager.TrafficManager/src/Customization/ProfileListResult.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.ResourceManager.TrafficManager.Models
+{
+    /// <summary>
+    /// Changing access modifier from internal to public since it's a result from public API.
+    /// </summary>
+    public partial class ProfileListResult
```

Comment:
is this still needed?

Reply:
yes, it changes default generated modifier from internal to public. The code does not compile without this customization.

Reply:
These objects shouldn't be exposed on public endpoints, lets try changing it back to internal and find out where the leak is and address that.